var/spack/repos/builtin/packages/asdf-cxx/package.py

```diff
@@ -45,5 +45,5 @@ class AsdfCxx(CMakePackage):
     # An error in the cmake script requires swig all the time, not only when
     # Python bindings are used
     depends_on('swig @3.0.0:3.999.999', type='build')
-    depends_on('yaml-cpp')
+    depends_on('yaml-cpp @0.6.3')
```

Comment:
Can you add a comment above here that this particular package _requires_ this exact version?

Reply:
Done.